server/src/main/java/org/elasticsearch/bootstrap/Elasticsearch.java

```diff
@@ -198,12 +199,16 @@ private static void initPhase2(Bootstrap bootstrap) throws IOException {
             VectorUtil.class
         );
 
-        // install SM after natives, shutdown hooks, etc.
-        org.elasticsearch.bootstrap.Security.configure(
-            nodeEnv,
-            SECURITY_FILTER_BAD_DEFAULTS_SETTING.get(args.nodeSettings()),
-            args.pidFile()
-        );
+        if (Boolean.parseBoolean(System.getProperty("es.entitlements.enabled", "false"))) {
```

Comment:
FWIW, this can be simplified as `Boolean.getBoolean("es.entitlements.enabled")`.

Reply:
Oh yes, in an earlier iteration, the default was `true`. But now this just looks dumb.

Reply:
Hey how about that.

```
Forbidden method invocation: java.lang.Boolean#getBoolean(java.lang.String) [use org.elasticsearch.core.Booleans#parseBoolean(java.lang.String)]
  in org.elasticsearch.bootstrap.Elasticsearch (Elasticsearch.java:202)
```

Reply:
🤦 Ah, I actually checked to see that we use this method, and we do, but only in tests, where apparently it is not forbidden.